### PR TITLE
feat(docs): add examples to site

### PR DESCRIPTION
**Pull request by calcsam**

### CODE REVIEW

File: docs/docs/adding-search-with-js-search.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Adding Search with JS Search
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/adding-search-with-js-search.md:4]:
```suggestion
  - label: Using js-search
```

File: docs/docs/building-an-ecommerce-site-with-shopify.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 ---
 title: Building an E-commerce site with Shopify
+examples:
+  - label: Gatsby Shopify Starter
+    href: "https://github.com/gatsbyjs/gatsby-starter-shopify"
+  - label: Using Shopify
```
Comment (KyleAMathews) [inline docs/docs/building-an-ecommerce-site-with-shopify.md:6]: probably just remove

File: docs/docs/conceptual/making-your-site-accessible.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Making Your Site Accessible
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/conceptual/making-your-site-accessible.md:4]:
```suggestion
  - label: Using react-skip-nav
```

File: docs/docs/how-to/adding-common-features/adding-an-rss-feed.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Adding an RSS Feed
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/adding-common-features/adding-an-rss-feed.md:4]:
```suggestion
  - label: Using RSS Feed
```

File: docs/docs/how-to/adding-common-features/adding-page-transitions-with-plugin-transition-link.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Adding Page Transitions with gatsby-plugin-transition-link
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/adding-common-features/adding-page-transitions-with-plugin-transition-link.md:4]:
```suggestion
  - label: Using page-transitions
```

File: docs/docs/how-to/adding-common-features/creating-a-sitemap.md [hunk 6]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Creating a Sitemap
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/adding-common-features/creating-a-sitemap.md:4]:
```suggestion
  - label: Using gatsby-plugin-sitemap
```

File: docs/docs/how-to/adding-common-features/processing-payments-with-stripe.md [hunk 7]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Processing Payments with Stripe
+examples:
+  - label: Source Repo
```
Comment (KyleAMathews) [inline docs/docs/how-to/adding-common-features/processing-payments-with-stripe.md:4]:
```suggestion
  - label: Using Stripe
```

File: docs/docs/how-to/custom-configuration/typescript.md [hunk 8]
```diff
@@ -1,5 +1,8 @@
 ---
 title: TypeScript and Gatsby
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/custom-configuration/typescript.md:4]:
```suggestion
  - label: Using Typescript
```

File: docs/docs/how-to/images-and-media/working-with-video.md [hunk 9]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Working with Video
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/images-and-media/working-with-video.md:4]:
```suggestion
  - label: Using video
```

File: docs/docs/how-to/plugins-and-themes/configuring-usage-with-plugin-options.md [hunk 10]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Configuring Plugin Usage with Plugin Options
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/plugins-and-themes/configuring-usage-with-plugin-options.md:4]:
```suggestion
  - label: Using plugin options
```

File: docs/docs/how-to/previews-deploys-hosting/path-prefix.md [hunk 11]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Adding a Path Prefix
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/previews-deploys-hosting/path-prefix.md:4]:
```suggestion
  - label: Using path-prefix
```

File: docs/docs/how-to/querying-data/using-gatsby-without-graphql.md [hunk 12]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Gatsby without GraphQL
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/querying-data/using-gatsby-without-graphql.md:4]:
```suggestion
  - label: Using Gatsby without GraphQL
```

File: docs/docs/how-to/routing/adding-markdown-pages.md [hunk 13]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Adding Markdown Pages
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/routing/adding-markdown-pages.md:4]:
```suggestion
  - label: Using gatsby-transformer-remark
```

File: docs/docs/how-to/routing/client-only-routes-and-user-authentication.md [hunk 14]
```diff
@@ -1,5 +1,10 @@
 ---
 title: Client-only Routes & User Authentication
+examples:
+  - label: Client-only Paths
+    href: "https://github.com/gatsbyjs/gatsby/examples/client-only-paths"
+  - label: Simple Auth
```
Comment (KyleAMathews) [inline docs/docs/how-to/routing/client-only-routes-and-user-authentication.md:6]:
```suggestion
  - label: Simple Authentication
```

File: docs/docs/how-to/routing/creating-routes.md [hunk 15]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Creating Routes
+examples:
+  - label: Collection Routing
```
Comment (KyleAMathews) [inline docs/docs/how-to/routing/creating-routes.md:4]:
```suggestion
  - label: Using Collection Routing
```

File: docs/docs/how-to/routing/mdx.md [hunk 16]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Add components to content using MDX
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/routing/mdx.md:4]:
```suggestion
  - label: Using MDX
```

File: docs/docs/how-to/sourcing-data/sourcing-from-contentful.md [hunk 17]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Sourcing from Contentful
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/sourcing-data/sourcing-from-contentful.md:4]:
```suggestion
  - label: Using Contentful
```

File: docs/docs/how-to/sourcing-data/sourcing-from-drupal.md [hunk 18]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Sourcing from Drupal
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/sourcing-data/sourcing-from-drupal.md:4]:
```suggestion
  - label: Using Drupal
```

File: docs/docs/how-to/sourcing-data/sourcing-from-wordpress.md [hunk 19]
```diff
@@ -1,5 +1,10 @@
 ---
 title: Sourcing from WordPress
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/sourcing-data/sourcing-from-wordpress.md:4]:
```suggestion
  - label: Using WordPress
```

File: docs/docs/how-to/sourcing-data/sourcing-from-wordpress.md [hunk 20]
```diff
@@ -1,5 +1,10 @@
...
+    href: "https://github.com/gatsbyjs/gatsby/examples/using-wordpress"
+  - label: Official Example w/ ACF
```
Comment (KyleAMathews) [inline docs/docs/how-to/sourcing-data/sourcing-from-wordpress.md:6]:
```suggestion
  - label: Using WordPress with ACF
```

File: docs/docs/how-to/styling/css-modules.md [hunk 21]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Component-Scoped Styles with CSS Modules
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/styling/css-modules.md:4]:
```suggestion
  - label: Using CSS Modules
```

File: docs/docs/how-to/styling/emotion.md [hunk 22]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Emotion
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/styling/emotion.md:4]:
```suggestion
  - label: Using Emotion
```

File: docs/docs/how-to/styling/sass.md [hunk 23]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Using Sass in Gatsby
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/styling/sass.md:4]:
```suggestion
  - label: Using Sass
```

File: docs/docs/how-to/styling/styled-components.md [hunk 24]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Styled Components
+examples:
+  - label: Official Exaample
```
Comment (KyleAMathews) [inline docs/docs/how-to/styling/styled-components.md:4]:
```suggestion
  - label: Using Styled Components
```

File: docs/docs/how-to/testing/end-to-end-testing.md [hunk 25]
```diff
@@ -1,5 +1,8 @@
 ---
 title: End-to-End Testing
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/testing/end-to-end-testing.md:4]:
```suggestion
  - label: Using Cypress
```

File: docs/docs/how-to/testing/unit-testing.md [hunk 26]
```diff
@@ -1,5 +1,8 @@
 ---
 title: Unit Testing
+examples:
+  - label: Official Example
```
Comment (KyleAMathews) [inline docs/docs/how-to/testing/unit-testing.md:4]:
```suggestion
  - label: Using Jest
```

**Review by KyleAMathews — COMMENTED** [review]
EXAMPLE ALL THE THINGS

File: docs/docs/creating-a-local-plugin.md [hunk 27]
```diff
@@ -1,5 +1,10 @@
 ---
 title: Creating a Local Plugin
+examples:
+  - label: Using Local Plugins
+    href: "https://github.com/gatsbyjs/gatsby/tree/master/examples/using-local-plugins"
+  - label: Using Multiple Local Plugins
+    href: "https://github.com/gatsbyjs/gatsby/examples/using-multiple-local-plugins"
```
Comment (calcsam) [inline docs/docs/creating-a-local-plugin.md:7]:
```suggestion
    href: "https://github.com/gatsbyjs/gatsby/tree/master/examples/using-multiple-local-plugins"
```

File: docs/docs/building-an-ecommerce-site-with-shopify.md [hunk 28]
```diff
@@ -1,5 +1,10 @@
...
+    href: "https://github.com/gatsbyjs/gatsby/tree/master/examples/using-shopify"
```
Comment (calcsam) [inline docs/docs/building-an-ecommerce-site-with-shopify.md:7]:
```suggestion
```

File: docs/docs/building-an-ecommerce-site-with-shopify.md (hunk 2 above)
Comment (calcsam) [inline docs/docs/building-an-ecommerce-site-with-shopify.md:6]:
```suggestion
```

**Review by KyleAMathews — APPROVED** [review]

**Pull request merged**